moped-editor/src/components/GridTable/Search.js

```diff
@@ -337,30 +196,6 @@ const Search = ({
           />
         </Paper>
       </Popper>
-      <Dialog
```

Comment:
Moved this up the tree into `ProjectListViewTable` since it doesn't have any dependencies in this component anymore.